cli/progress/progress.go

```diff
@@ -62,7 +62,7 @@ func NewMultiBar(id string, data progressData) *progressbar {
 		id:           id,
 		maxNameWidth: maxNameWidth,
 		Data:         data,
-		isTTY:        isatty.IsTerminal(os.Stdout.Fd()) && false, // FIXME: re-enable the detection
+		isTTY:        isatty.IsTerminal(os.Stdout.Fd()),
```

Comment:
Do we still support piping args into `cnspec`? If not, maybe we can remove some code here (lines 84-96):
```
// else if not a TTY (but piped?)
else {
		go func() {
			for {
				time.Sleep(time.Second / progressPipedFps)
				if p.Data.complete {
					break
				}
				fmt.Print(p.View() + "\r\033[2A")
			}
			fmt.Print(p.View())
		}()
	}
```

Reply:
Good point, lets revisit that if we can simplify the approach here